specification/glossary.md

```diff
@@ -1,11 +1,129 @@
 # Glossary
 
-*This section is non-normative.*
+## <a name="glossary:adjustment">Adjustment</a>
 
-`TODO` Needs to be completed and linked
+A charge representing a modification to billing data to account for certain events or circumstances not previously captured, or captured incorrectly. Examples include: billing errors, service disruptions, or pricing changes.
 
-Resource:
-> Add definition
+## <a name="glossary:amortization">Amortization</a>
 
-Service:
-> Add definition
+The distribution of upfront costs over time to accurately reflect the consumption or benefit derived from the associated resources or services. Amortization is valuable when the commitment period (time duration of the cost) extends beyond the granularity of the source report.
+
+## <a name="glossary:availability-zone">Availability Zone</a>
```

Comment:
I was trying to avoid having the glossary entries show up with link formatting. Let me see if I can mess with the CSS but this doesn't look good IMO. It looks like it's a link to something else when we're trying to just drop an anchor to this term. That's why I tried to just put the link around the * (empty a-tag) doesn't work on PDF, it works on the HTML rendering though.